packages/treat/tests/fixtures/react/Demo.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
```

Comment:
Not your fault but this file is no longer used. Just remove it.

Reply:
Was sure I did this... but I guess not 🤷‍♂